front/lib/resources/group_resource.ts

```diff
@@ -129,10 +129,6 @@ export class GroupResource extends BaseResource<GroupModel> {
     if (key.isSystem) {
       whereCondition = {
         ...whereCondition,
-        [Op.or]: [
```

Comment:
Actually not sure I understand the diff here can you explain a bit more?

Reply:
I'm removing the constraint on kind, so that we get all groups of the workspace, not only system or global

Reply:
Ok I see that makes sense.

Reply:
Can you fix/update the comment above :pray: